test/instances/ezETH/EzETHHyperdrive.t.sol

```diff
@@ -54,187 +52,78 @@ contract EzETHHyperdriveTest is HyperdriveTest {
     // mocking.  To test with their deployed code we use a shorter position
     // duration.
     uint256 internal constant POSITION_DURATION_2_WEEKS = 15 days;
-    address internal ETH_WHALE = 0x00000000219ab540356cBB839Cbe05303d7705Fa;
-    address internal EZETH_WHALE = 0x40C0d1fbcB0A43A62ca7A241E7A42ca58EeF96eb;
     uint256 internal constant STARTING_BLOCK = 19119544;
 
-    HyperdriveFactory factory;
-    address deployerCoordinator;
+    // Whale accounts.
+    address internal EZETH_WHALE = 0x40C0d1fbcB0A43A62ca7A241E7A42ca58EeF96eb;
+    address[] internal whaleAccounts = [EZETH_WHALE];
+
+    // The configuration for the integration testing suite.
+    IntegrationConfig internal __testConfig =
+        IntegrationConfig(
+            whaleAccounts,
+            IERC20(EZETH),
+            IERC20(ETH),
+            1e6,
+            1e15,
+            POSITION_DURATION_2_WEEKS,
+            false
+        );
+
+    constructor() IntegrationTest(__testConfig) {}
 
     function setUp() public override __mainnet_fork(STARTING_BLOCK) {
+        // Depositing with ETH is not allowed for this pool so we need to get
+        // some ezETH for alice first.
+        vm.startPrank(EZETH_WHALE);
+
+        vm.deal(EZETH_WHALE, 50_000e18);
+        RESTAKE_MANAGER.depositETH{ value: 40_000e18 }();
```

Comment:
What's up with these amounts? Why are we using 50_000e18 for one and 40_000e18 for the other?